project/src/test/java/com/google/sps/data/PlacesFetcherTest.java

```diff
@@ -125,13 +132,20 @@ public void fetch_zeroSearchResults_returnsEmptyList() throws Exception {
   }
 
   @Test
-  public void fetch_validSearchResults_returnsListOfPlaces()
-      throws ApiException, InterruptedException, IOException {
+  public void fetch_validSearchResults_returnsListOfPlaces() throws Exception {
```

Comment:
Shouldn't it throw a FetcherException?

Reply:
I think that in tests it's good practice to throw `Exception`, this way you don't need to specify each exception that might be thrown in the process.

Reply:
Cool, good to know :)